spec/filtering_spec.rb

```diff
@@ -101,6 +101,29 @@ def self.name
         expect(Graphiti::Util::Hash).to_not receive(:split_json)
         expect(records.map(&:id)).to eq([employee2.id, employee3.id])
       end
+
+      context "and it is a complex hash" do
```

Comment:
Looks like we have two tests with the same description - maybe "deeply nested hash"?

Reply:
Done!